src/main/java/uk/gov/pay/webhooks/deliveryqueue/dao/WebhookDeliveryQueueEntity.java

```diff
@@ -75,7 +75,8 @@ public Instant getCreatedDate() {
     public enum DeliveryStatus {
         PENDING,
         SUCCESSFUL,
-        FAILED
+        FAILED,
+        WONT_SEND
```

Comment:
I wonder if `WILL_NOT_SEND` might get around the inability to include apostrophes (compounded by the fact that ‘wont’ is an actual word).